pytorch_lightning/callbacks/progress/tqdm_progress.py

```diff
@@ -115,6 +115,26 @@ def __getstate__(self) -> Dict:
         # can't pickle the tqdm objects
         return {k: v if not isinstance(v, _tqdm) else None for k, v in vars(self).items()}
 
+    @property
+    def sanity_check_description(self) -> str:
+        return "Validation Sanity Check"
+
+    @property
+    def train_description(self) -> str:
```

Comment:
This customization could be left for a separate PR

Reply:
just trying to avoid duplicate manual text at 2 different places.

Reply:
Fine